src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -207,21 +208,23 @@ export class GcpCreateServerApp extends LitElement {
   private renderBillingAccountSetup() {
     return html`
       <outline-step-view id="billingAccountSetup" display-action="">
-        <span slot="step-title">Activate your Google Cloud Platform account.</span>
-        <span slot="step-description">Enter your billing information on Google Cloud Platform.</span>
+        <span slot="step-title">Add billing information to your Google Cloud Platform account.</span>
+        <span slot="step-description">Open the Cloud Console billing page and add an account in order to proceed.</span>
         <span slot="step-action">
-          <paper-button id="createServerButton" @tap="${this.handleBillingVerificationNextTap}">
-            NEXT
+          <paper-button id="openBillingPage" @tap="${this.openBillingPage}">
```

Comment:
Maybe a way to go about this is to automatically start the polling, show "OPEN" when the polling is happening, and "NEXT" so the user can move on when the polling stops.  Otherwise there will be a gap where the user is stuck waiting for the next poll.

Reply:
That was the purpose of the REFRESH button.  I changed the name to "CHECK AGAIN" to try to make that clearer:

![Screenshot from 2021-06-04 12-17-02](https://user-images.githubusercontent.com/191945/120834121-9c850f00-c530-11eb-88ea-4cbfe5ade8d7.png)